File: MathCalculations/Dart/FibonacciSeq/Fibonacci.dart
int fibonacciRecursive(int n) {
  return (n <= 1) ? n : fibonacciRecursive(n - 1) + fibonacciRecursive(n - 2);
}

void main() {
  final index = 35; 
  final stopwatch = Stopwatch()..start();
  final fibonacciResult = fibonacciRecursive(index);
  stopwatch.stop();
  double elapsedTime = stopwatch.elapsedMilliseconds.toDouble();
  String formattedTime = elapsedTime.toStringAsFixed(2);
  print(formattedTime);

  //print('Fibonacci sequence at index $index: $fibonacciResult');
}

File: MathCalculations/Kotlin/FibonacciSeq/Fibonacci.kt
fun fibonacciRecursive(n: Int): Long {
    return if (n <= 1) {
        n.toLong()
    } else {
        fibonacciRecursive(n - 1) + fibonacciRecursive(n - 2)
    }
}

fun main() {
    val stopwatch = System.currentTimeMillis()
    val n = 35
    val fibonacciResult = fibonacciRecursive(n)

    var elapsedTime = System.currentTimeMillis() - stopwatch
    var formattedTime = String.format("%.2f", elapsedTime.toDouble())
    formattedTime = formattedTime.replace(',', '.')
    println("$formattedTime")
    //println("Fibonacci sequence at index $n: $fibonacciResult")
}
